velox/exec/SpillOperatorGroup.h

```diff
@@ -141,6 +152,9 @@ class SpillOperatorGroup {
   // The participating spill operators which won't change after starts the
   // group.
   std::vector<Operator*> operators_;
+  // Used to hold the shared reference on the associated drivers to keep the
+  // operators alive while running spilling.
+  std::vector<std::weak_ptr<Driver>> drivers_;
```

Comment:
Curious, instead of adding weak_ptr to Driver, could we promote the raw pointer of Operator* to weak_ptr? Looking from the description it is the operator that is referenced that caused the crash. Thanks

Reply:
We can't. I don't think weak_ptr take raw point ctor. Operator is owned by driver through unique_ptr.